### PR TITLE
Repo layer

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-498

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/data/CallbackResultRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.data.callbackresult.RequestType.ATTACH_TO_CASE;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.data.callbackresult.RequestType.CREATE_CASE;
+
+@ActiveProfiles({"nosb", "db-test"})
+@IntegrationTest
```
Comment (mdayican) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/data/CallbackResultRepositoryTest.java:19]: you can use  @DataJpaTest

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/data/CallbackResultRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+public class CallbackResultRepositoryTest {
+
+    public static final String ER_ID_1 = "ER_ID_1";
+    public static final String CASE_ID_1 = "CASE_ID_1";
+    @Autowired private CallbackResultRepository repo;
+    @Autowired private DbHelper dbHelper;
+
+    @BeforeEach
+    void setUp() {
+        dbHelper.deleteAll();
```
Comment (mdayican) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/data/CallbackResultRepositoryTest.java:29]: do not need if you use @DataJpaTest

**Review by mdayican — APPROVED** [review]

File: src/integrationTest/resources/application.properties [hunk 3]
```diff
@@ -31,3 +31,6 @@ service-config.services[0].update-url=http://localhost:${wiremock.server.port}/u
 service-config.services[0].transformation-url=http://localhost:${wiremock.server.port}/transform-exception-record
 
 flyway.skip-migrations=false
+
+spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
+spring.datasource.url=jdbc:tc:postgresql:11.0://localhost/blob_router
```
Comment (mdayican) [inline src/integrationTest/resources/application.properties:36]: the name at the end is wrong

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -93,7 +93,7 @@ withPipeline(type, product, component) {
     builder.gradle('integration')
   }
 
-  enableDbMigration('bulk-scan')
+  enableDbMigration('bs_orchestrator')
```
Comment (mdayican) [inline Jenkinsfile_CNP:96]: it is vault name, it should be 'bulk-scan'

File: src/integrationTest/resources/application.properties (hunk 3 above)
Comment (mdayican) [reply src/integrationTest/resources/application.properties:36]: and you need to add this to functional test application yaml. it should solve the problem.

Comment (mdayican) [reply src/integrationTest/resources/application.properties:36]: and you need to add this to functional test application yaml. it should solve the problem.

Comment (mdayican) [inline src/integrationTest/resources/application.properties:36]: and you need to add this to functional test application yaml. it should solve the problem.

**Comment by apereverzin** [comment]
Will re-create

**Pull request closed without merging**